kordx-commands-runtime/src/main/kotlin/com/gitlab/kordlib/kordx/commands/argument/text/Extensions.kt

```diff
@@ -18,10 +19,10 @@ fun <CONTEXT> Argument<String, CONTEXT>.whitelist(
             when {
                 ignoreCase -> when {
                     whitelist.any { word -> word.equals(it, true) } -> FilterResult.Pass
-                    else -> FilterResult.Fail("expected one of $whitelist (not case sensitive) but got $it")
+                    else -> FilterResult.Fail("expected one of ${whitelist.joinToString(", ")} (not case sensitive) but got $it")
                 }
                 it in whitelist -> FilterResult.Pass
-                else -> FilterResult.Fail("expected one of $whitelist (case sensitive) but got $it")
+                else -> FilterResult.Fail("expected one of ${whitelist.joinToString(", ")} (case sensitive) but got $it")
```

Comment:
Please use `contentToString` instead for a list-like string representation.

Reply:
Wasn't sure what looks better